rest_framework_simplejwt/locale/ro_RO/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,113 @@
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR Daniel Cuznetov <danielcuznetov04@gmail.com>, 2022.
+msgid ""
+msgstr ""
+"Project-Id-Version: djangorestframework_simplejwt\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-07-13 10:45+0100\n"
+"Last-Translator: Daniel Cuznetov <danielcuznetov04@gmail.com>\n"
+"Language: ro_RO\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+
+#: authentication.py:78
+msgid "Authorization header must contain two space-delimited values"
+msgstr "Header de autorizare trebuie să conțină două valori separate de spațiu"
```

Comment:
"Header-ul(antetul) de autorizare trebuie să conțină două valori separate prin spațiu"